ui/src/organizations/components/Members.tsx

```diff
@@ -109,8 +117,28 @@ export default class Members extends PureComponent<Props, State> {
   }
 
   private async getUsers() {
+    const {members} = this.props
+
     const data = await client.users.getAllUsers()
-    this.setState({users: data.users})
+
+    const users = {}
+
+    for (const key of data.users) {
```

Comment:
We could make this functional using `data.users.forEach`

Reply:
Yes, I updated it